content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -300,6 +300,31 @@ spec:
         value: http,nestjs-core # comma-separated list of the instrumentation package names without the `@opentelemetry/instrumentation-` prefix.
 ```
 
+To keep all default libraries and disable only specific instrumentation
+libraries you can use the `OTEL_NODE_DISABLED_INSTRUMENTATIONS` environment
+variable. For details, see
+[Excluding instrumentation libraries](/docs/zero-code/js/configuration/#excluding-instrumentation-libraries).
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+# ... other fields skipped from this example
+spec:
+  # ... other fields skipped from this example
+  nodejs:
+    env:
+      - name: OTEL_NODE_DISABLED_INSTRUMENTATIONS
+        value: fs,grpc # comma-separated list of the instrumentation package names without the `@opentelemetry/instrumentation-` prefix.
+```
+
+{{% alert title="Notes" color="info" %}}
+
+Note that `OTEL_NODE_ENABLED_INSTRUMENTATIONS` and
+`OTEL_NODE_DISABLED_INSTRUMENTATIONS` are mutually exclusive. If both are set,
+all instrumentations are disabled.
```

Comment:
Are you sure this is correct? My understanding is that if both are set, then first `*_ENABLED_*` is used to establish a starting set, then any of those listed in `*_DISABLED_*` are removed.

```
% cd ~/src/opentelemetry-js-contrib/metapackages/auto-instrumentations-node
% node
> var mod = require('./')
undefined
> process.env.OTEL_NODE_ENABLED_INSTRUMENTATIONS = 'dns,fs,hapi'
'dns,fs,hapi'
> mod.getNodeAutoInstrumentations().map(instr => instr.instrumentationName)
[
  '@opentelemetry/instrumentation-dns',
  '@opentelemetry/instrumentation-fs',
  '@opentelemetry/instrumentation-hapi'
]
> process.env.OTEL_NODE_DISABLED_INSTRUMENTATIONS = 'bunyan,fs'
'bunyan,fs'
> mod.getNodeAutoInstrumentations().map(instr => instr.instrumentationName)
[
  '@opentelemetry/instrumentation-dns',
  '@opentelemetry/instrumentation-hapi'
]
```

Perhaps:

```suggestion
If both environment variables are specified, `OTEL_NODE_ENABLED_INSTRUMENTATIONS` is first used to establish a set of instrumentation package names, then any names from `OTEL_NODE_DISABLED_INSTRUMENTATIONS` are removed from that set. Any remaining instrumentation names are enabled.
```

Reply:
Yeah no you're totally correct, I'm not sure what I was doing when I wrote the original test. Will update there and then here

Reply:
Updated!